internal/build/lifecycle_execution.go

```diff
@@ -655,7 +718,7 @@ func (l *LifecycleExecution) Export(ctx context.Context, buildCache, launchCache
 	flags := []string{
 		"-app", l.mountPaths.appDir(),
 		"-cache-dir", l.mountPaths.cacheDir(),
-		"-stack", l.mountPaths.stackPath(),
+		// "-stack", l.mountPaths.stackPath(), // TODO: remove, replace with "-run" as part of https://github.com/buildpacks/pack/issues/1301
```

Comment:
probably  you meant to do some resolution here

Reply:
Updated in e8c869a7